arch/src/aarch64/layout.rs

```diff
@@ -20,6 +20,13 @@
 //           |                            DRAM                               |
 //           |                                                               |
 //           |                                                               |
+// 4GB       +---------------------------------------------------------------+
```

Comment:
Since you are touching this part, could you please do us a favor to change the text at the top of the layout: `// Memory layout of Aarch64 guest:` to `// Memory layout of AArch64 guest:`?

Thank you very much!

Reply:
Sure.